deploy/kubernetes/setup.sh

```diff
@@ -4,7 +4,7 @@ set -e
 usage() {
   echo
   echo 'Usage:'
-  echo '  setup.sh [-c collector-name] [-k cluster-name] [-n namespace] [-a useAlpha] <endpoint> <access-id> <access-key>'
+  echo '  setup.sh [-c collector-name] [-k cluster-name] [-n namespace] [-a use-alpha] [-d deploy] [-y download-yaml] <endpoint> <access-id> <access-key>'
```

Comment:
technically changing parameter name is a breaking change in case anyone has integrated this script into their automation, but since it's so new it's probably safe to change to `use-alpha` now.

Reply:
it is actually not the parameter name, just a placeholder for the parameter value. the name is just `-a`

Reply:
oh good point.